src/main/java/handler/BalanceHandler.java

```diff
@@ -0,0 +1,12 @@
+package handler;
+
+import db.Storage;
+import model.FruitTransaction;
+import strategy.TransactionHandler;
+
+public class BalanceHandler implements TransactionHandler {
+    @Override
+    public void handler(FruitTransaction transaction) {
```

Comment:
```suggestion
    public void handle(FruitTransaction transaction) {
```
method names should be verbs